_maps/map_files/stations/boxstation.dmm

```diff
@@ -5354,8 +5354,7 @@
 /area/station/legal/lawoffice)
 "asO" = (
 /obj/machinery/computer/prisoner{
-	dir = 4;
```

Comment:
Why did this (and many changes below) remove req_access from these prisoner computers and bar closets?